tests/Person.spec.ts

```diff
@@ -58,171 +64,254 @@ test.describe(`Testfälle für die Administration von Personen: Umgebung: ${proc
-      const responseBody_schulstrukturknoten = await response_schulstrukturknoten.json();
+      const response_schulstrukturknoten = await page.waitForResponse(
+        (response) =>
+          response.url().includes("/api/personenkontext/schulstrukturknoten")
+      );
+      const responseBody_schulstrukturknoten =
+        await response_schulstrukturknoten.json();
       let gueltig = true;
-      
-      (responseBody_schulstrukturknoten.moeglicheSsks).forEach(element => {
-        if(!(element.typ === TYP_ORGA_SCHULE)) {
+
+      responseBody_schulstrukturknoten.moeglicheSsks.forEach((element) => {
+        if (!(element.typ === TYP_ORGA_SCHULE)) {
           gueltig = false; // element hat eine Organisation mit einem ungültigen Typ
           expect(gueltig).toBe(true); // Der Testfall wird auf failed gesetzt
         }
-      });   
-    })
+      });
+    });
 
     await test.step(`Rolle SuS auswaehlen und verfügbare Einträge für die Organisationsebene prüfen(Gültige Organisationen: SCHULE)`, async () => {
       await PersonCreationView.combobox_Rolle_Clear.click();
       await PersonCreationView.combobox_Rolle.click();
       await page.getByText(Rolle_SUS).click();
 
-      const response_schulstrukturknoten = await page.waitForResponse((response) => response.url().includes("/api/personenkontext/schulstrukturknoten"));
-      const responseBody_schulstrukturknoten = await response_schulstrukturknoten.json();
+      const response_schulstrukturknoten = await page.waitForResponse(
+        (response) =>
+          response.url().includes("/api/personenkontext/schulstrukturknoten")
+      );
+      const responseBody_schulstrukturknoten =
+        await response_schulstrukturknoten.json();
       let gueltig = true;
-      
-      (responseBody_schulstrukturknoten.moeglicheSsks).forEach(element => {
-        if(!(element.typ === TYP_ORGA_SCHULE)) {
+
+      responseBody_schulstrukturknoten.moeglicheSsks.forEach((element) => {
+        if (!(element.typ === TYP_ORGA_SCHULE)) {
           gueltig = false; // element hat eine Organisation mit einem ungültigen Typ
           expect(gueltig).toBe(true); // Der Testfall wird auf failed gesetzt
         }
-      });   
-    })
-  })
-})
\ No newline at end of file
+      });
+    });
+  });
+
+  test("In der Ergebnisliste die Suchfunktion ausführen", async ({ page }) => {
+    const Startseite = new StartPage(page);
+    const Menue = new MenuPage(page);
+    const PersonManagementView = new PersonManagementViewPage(page);
+
+    await test.step(`Benutzerverwaltung öffnen und Suche nach Vornamen `, async () => {
+      await Startseite.card_item_schulportal_administration.click();
+      await Menue.menueItem_AlleBenutzerAnzeigen.click();
+      await expect(PersonManagementView.text_h2_Benutzerverwaltung).toHaveText(
+        "Benutzerverwaltung"
+      );
+      await PersonManagementView.input_Suchfeld.fill("Yvonne");
```

Comment:
Ist sichergestellt, dass dieser Vorname immer vorhanden ist? Sollte sich am Seeding was ändern, müssen die Tests angepasst werden. Sicherer wäre es, wenn für die Suche ein extra Benutzer im Test angelegt würde. Wie wird das auf Prod laufen zukünftig?

Reply:
Mittlerweile geht das Löschen von Benutzern. Das Löschen und Anlage von Benutzern kommt in SPSH-689.